lib/cache/Cache.js

```diff
@@ -44,14 +45,33 @@ class Cache {
         return loc
     }
 
+    async write(filepath, data) {
+        const loc = this.getCacheLocation(filepath)
+        await mkdirp(path.dirname(loc))
+        try {
+            fs.writeFileSync(loc, data)
```

Comment:
You can do ‘await fs.writeFile’ with the ‘fs-extra’ lib which is consistent with ‘mkdirp’ above, though in practice they do the same thing here.